### DISCUSSION TITLE
Public forums closure

### DISCUSSION THREAD

**Discussion by valinet** [valinet/ExplorerPatcher#1068] [Announcements]
Hi

In the light of how things have been evolving lately, I have taken the decision to limit the interactions on this repository. This means that Discussions, Issues, commit messages etc will soon become read only until further notice. 

The decision is motivated by a couple of factors, which I will briefly summarize below:

* Failure of attracting a developer audience for this project. Since I am still the only developer on this project, I don't really need all this infrastructure to keep track of current work; it gets in the way more than it helps.
* These forums largely becoming anything but avenues for development. Although GitHub is a site primarily geared towards developers, approx. 99.99% of threads opened here are support tickets, feature requests, stuff marginally related to the project, ranting, trash talking etc. My vision was never to have this infrastructure solely as a support platform - I believe that can take place better in other places.
* Difficulty in maintaining this project focused. Feedback and suggestions are great, but too much of that makes it difficult to manage growth and prioritize. This doesn't mean, though, that new features won't be added, but as always, the best way to have something desperately wanted now is to fork the project and implement the functionality yourself.
* Constant disregard of community rules and application material (wiki, guides, README). This forums were never meant as a shortcut for skipping the README, ignoring the wiki, disregarding the written guides etc. Multiple "me too" posts were all too often made, despite the "issue' being sorted out already (like, EP not being tested on Insider Preview, and generally not willing to do so), or sorted in a similar thread.
* My personal desire to spend a more controlled amount of time with this project. As some of you may have noticed (or not), this project has been eating a lot of my time, especially recently, but really, all the way since I started it. The journey's been great, it has taught me tons of stuff, but I really think it's time to move on, dedicate more time on awesome new stuff, concentrate more on my other different works etc. Of course, I am still using Windows 11 daily, so I am keen on using and having EP work the way it was envisioned, since I have come to depend on its functionalities daily.

What about the source? The source code will, of course, still remain fully public. What really changes with this is a shift in development model, from open development, to an AOSP-like model, where stuff is developed based on how I envision this application, behind closed doors, and the code is updated whenever changes happen, usually along new versions. Similarly, development won't stop. I don't know what your expectations are, but please refrain from calling the project "dead" and the likes. Considering this, please refrain from messaging me with feature requests, as, with all due respect, I really don't have time for any anymore.

What's next? I will close the forums here some time after the next major release drops, which should be today or tomorrow, unless something major happens (again). As you may have noticed, this latest release seems to never make it, having by far the most number of pre-releases, with almost 2 months since the latest release. On a brighter note, it's true that a lot of functionalities have been implemented since the latest release, and of course those take time to polish an get ready. Hopefully, with it finally being in the state I kind of envisioned it for shipping, it will make it to a release in the coming days.

Where to go for support? I don't necessarily have a recommendation for that. I have seen people discussing EP here, for example: https://www.elevenforum.com/t/new-version-of-explorerpatcher.1991/

Of course, I am thankful for the members of the community here that took the time to help and assist others, and tried to help keep the discussions on track.

The closure here is not (may not be) permanent, but will last until further notice.

Thank you very much and I hope EP serves you well.

@valinet

**Comment by OrthodoxWindows** [comment]
"Similarly, development won't stop. I don't know what your expectations are, but please refrain from calling the project "dead" and the likes. "

You are right to insist on this point. Many people have abusively tell that ClassicShell, since its resumption by the community, was "dead", is it to turn many users, while OpenShell (ClassicShell) is still alive, and remains the best classic start menu for Windows (Without even talking about the toolbar for the file explorer).

Personally, if I fall on anyone who insinuates that ExplorerPatcher is "dead", I will repeat tirelessly the truth.

**Reply by valinet** [reply]
Yeah, that is very infuriating to hear about some project, especially when you have worked on and know hat it takes. No one has a duty to work on these projects, everyone that does, does so voluntarily and in whatever time they have. And no one stops all these dudes calling a project dead from picking themselves the thing up and reviving it. But yeah, it's always easier to criticize something than to get hands on with it. Thanks for the support.
